flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -358,6 +359,18 @@
                                     .list(text("'ng': new generation scheduler"))
                                     .build());
 
+    @Documentation.Section(Documentation.Sections.EXPERT_SCHEDULING)
+    public static final ConfigOption<SchedulerExecutionMode> SCHEDULER_EXECUTION_MODE =
+            key("execution-mode")
```

Comment:
Isn't this option a bit misleading with our batch/streaming execution mode? 

https://ci.apache.org/projects/flink/flink-docs-release-1.12/concepts/glossary.html#runtime-execution-mode

Wouldn't `scheduler-mode` be more specific to avoid confusion?

Reply:
Thanks a lot for your feedback. I will rename it to `scheduler-mode`.